lib/osf-components/addon/components/search-page/styles.scss

```diff
@@ -1,78 +1,55 @@
-.search-page {
-    background-color: $color-bg-gray;
-}
-
-.heading-wrapper {
-    text-align: center;
-    color: $color-text-white;
+// stylelint-disable max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type
+.heading-wrapper-mobile {
+    background-image: url('images/default-brand/bg-dark.jpg');
     background-color: $osf-dark-blue-navbar;
-}
-
-.provider-logo {
-    background: var(--hero-logo-img-url) top center no-repeat;
-    background-size: contain;
-    height: 70px;
-    width: 100%;
-    margin: 2em 0 1em;
-}
-
-.hero-overlay {
+    color: $color-text-white;
     display: flex;
     flex-direction: column;
-    width: 100%;
-    height: 100%;
-    min-width: 100%;
-    min-height: 100%;
-    position: relative;
-    background-color: transparent; // override .heading-wrapper background-color
+    align-items: end;
+    padding: 0;
 
-    &::after {
-        position: absolute;
-        top: 0;
-        left: 0;
+    label {
+        margin-left: 0.5rem;
+        padding: 0;
+        text-align: left;
         width: 100%;
-        height: 100%;
-        background: var(--hero-background-img-url);
-        background-size: cover;
-        z-index: -1;
-        content: '';
-    }
-}
 
-.heading-wrapper-institutions {
-    background-color: $color-shadow-gray-dark;
-    color: $color-text-gray-cool;
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    padding-bottom: 0.5rem;
+        > h1 {
+            font-weight: 500;
+            margin: 0.2rem;
+            padding: 0.75rem 0 0.5rem 0.5rem;
+        }
+    }
 
-    label {
-        max-width: 835px;
-        min-width: 250px;
-        width: 50vw;
-        padding-bottom: 0;
+    span {
+        display: flex;
+        flex-direction: row;
+        justify-content: flex-start;
+        margin-left: 0.5rem;
+        padding: 0 0 0.5rem 0.5rem;
+        width: 100%;
 
-        img {
-            max-height: 65px;
+        button:nth-of-type(2) {
+            transform: scale(0.5);
+            border-radius: 50%;
+            margin-left: 0;
+            position: absolute;
+            right: 0.1rem;
         }
+    }
 
-        p {
-            font-weight: 400;
-            text-align: center;
-            padding: 1rem 0;
-            margin: 0;
-        }
+    > button {
+        width: 100%;
     }
 }
 
 .heading-wrapper-institutions-mobile {
```

Comment:
Pending the institutions reverted PR: https://github.com/CenterForOpenScience/ember-osf-web/pull/1935 this and other desktop stylings for institutions can be removed, but can further discuss at scrum.